estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
You are missing a line at the end of the file. You can configure your IDE to do that automatically. This comment applies to all files in your PR.